deploy/helm/sumologic/values.yaml

```diff
@@ -2987,7 +2993,20 @@ otelcol:
                 - source/systemd
                 - batch
               exporters:
-                - sumologic/systemd
+                - sumologic
+            logs/kubelet:
+              receivers:
+                - fluentforward
+              processors:
+                - memory_limiter
+                - filter/include_fluent_tag_host
+                - attributes/remove_fluent_tag
+                - filter/include_kubelet
```

Comment:
Actually as I think of it now I believe we can swap those 2 to save processing of "non kubelet" entries, i.e. no need to remove the `fluent.tag` from a record that will be filtered out anyway.

The same can be done for systemd

Reply:
so we should move it as far as possible, we have additional filtering in sourceprocessor for example

Reply:
What do you mean "as far as possible"? Shouldn't we filter out as soon as possible in the pipeline?

Reply:
I mean moving `- attributes/remove_fluent_tag` just before `batch`